### PR TITLE
DTFS2-4054 : ACBS Interaction activities

### PR DESCRIPTION
**Pull request by abhi-markan**
ACBS `deal` and `facility` records creations are aided with durable functions response. Earlier ACBS activities were being created with-in ACBS library. This now has been moved to TFM due to inconsistency during the runtime. This commit deals with following updates and improvements:

* Activity API, Constant and Central API library has been removed from ACBS.
* `tfm-update-deal-controller.js` now decapsulate new activities.
* Addition of `console.error` when making an external call.
* Addition of a new TFM AIN GEF deal in mock deals.
* All activities objects are created in a single call.
* Improved documentation.
* Lint fixes

### CODE REVIEW

File: trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+      },
+      smeType: 'MICRO',
+      status: 'Completed',
+      updatedAt: 1643727452018,
+    },
+    facilities: [
+      {
+        _id: { $oid: '61f94a4327c1a7009cde1b9e' },
+        dealId: { $oid: '61f94a2427c1a7009cde1b9d' },
+        type: 'Cash',
```
Comment (IgnatG) [inline trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js:134]: constant

File: trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+      ],
+      id: '9',
+      mga: ['mga_ukef_1.docx', 'mga_ukef_2.docx'],
+      name: 'UKEF test bank (Delegated)',
+      partyUrn: '00318345',
+    },
+    bankInternalRefName: 'abc',
+    checkerId: '61f29adb6851c10012604bd1',
+    createdAt: 1643727396847,
+    dealType: 'GEF',
```
Comment (IgnatG) [inline trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js:20]: constant

File: trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+      {
+        role: 'checker',
+        userName: 'BANK1_CHECKER1',
+        createdAt: 1643735218306,
+        comment: '123123',
+      },
+    ],
+    eligibility: {
+      criteria: [
+        {
```
Comment (IgnatG) [inline trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js:32]: do we need all of these properties in here?

File: trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+        facilityType: '',
+        ukefFacilityId: '',
+        maker: '',
+        checker: '',
+      },
+    ],
+    status: 'Submitted',
+    submissionCount: 1,
+    submissionDate: '1643727452025',
+    submissionType: 'Automatic Inclusion Notice',
```
Comment (IgnatG) [inline trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js:207]: constant

Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js]: Updated.

File: trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js (hunk 3 above)
Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js]: Removed

File: trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js (hunk 2 above)
Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js]: Updated

File: trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js (hunk 1 above)
Comment (abhi-markan) [reply trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js]: Updated

File: trade-finance-manager-api/src/v1/helpers/activity.api-test.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  it('Shoule return label case for specified string', () => {
+    expect(labelCase('Automatic Inclusion Notice')).not.toEqual('Automatic Inclusion Notice');
+  });
+});
+
+describe('getTimestamp()', () => {
+  it('Shoule return epoch timestamp without milliseconds', () => {
+    expect(getTimestamp(mockDealRecord)).toEqual(1643629694);
+  });
+  it('Shoule return epoch timestamp without milliseconds', () => {
```
Comment (ttbarnes) [inline trade-finance-manager-api/src/v1/helpers/activity.api-test.js:41]: Shuole should be should 😉

File: trade-finance-manager-api/src/v1/helpers/activity.js [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+ * ACBS interaction records
+ * @param {Object} acbs Durable function output
+ * @returns {Array} An array of activities object
+ */
+const add = async (acbs) => {
+  if (acbs) {
+    acbsResponse = acbs;
+    const deal = await api.findOneDeal(acbs.portalDealId);
+    if (deal) {
+      return getActivities(deal);
```
Comment (ttbarnes) [inline trade-finance-manager-api/src/v1/helpers/activity.js:135]: No test for this?

**Review by ttbarnes — COMMENTED** [review]
LGTM - couple of comments but happy to merge and address later.

**Review by ttbarnes — APPROVED** [review]

**Pull request merged**